Makefile

```diff
@@ -13,4 +13,4 @@ docs: ## Generate projects documentation (from "Documentation" directory)
 test-docs: ## Test the documentation rendering
 	mkdir -p Documentation-GENERATED-temp
 
-	docker run --rm --pull always -v "$(shell pwd)":/project -t ghcr.io/typo3-documentation/render-guides:latest --config=Documentation --no-progress --fail-on-log
+	docker run --rm --pull always -v "$(shell pwd)":/project -t ghcr.io/typo3-documentation/render-guides:latest --config=Documentation --no-progress --fail-on-log --output-format=html
```

Comment:
Why in HTML? This has also nothing to do with conval, if I am not wrong.

Reply:
When you are running the test we have 3 rendering methods. 

So errors that are output during rendering are output 3 times. I will move this to another PR however.